contracts/modules/v2/CoverMigrator.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity ^0.8.9;
+
+import "../../interfaces/INXMToken.sol";
+import "../../interfaces/IMemberRoles.sol";
+import "../../interfaces/IPool.sol";
+import "../../interfaces/ICover.sol";
+import "../../interfaces/IIndividualClaims.sol";
+import "../../interfaces/IAssessment.sol";
+import "../../interfaces/IERC20Detailed.sol";
+import "../../interfaces/ICoverNFT.sol";
+
+import "../../abstract/MasterAwareV2.sol";
+
+/// Provides a way for contracts which don't use the Gateway.sol contract to migrate covers to V2
+/// using the submitClaim claim function.
+contract CoverMigrator is MasterAwareV2 {
```

Comment:
is this replacing an existing contract?

Reply:
Yes, Claims.sol. It's not explicitly mentioned in the comment above but it should be obvious. Some contracts use Claims.sol directly instead of Gateway.sol so we're leaving this CoverMigrator contract in place (CL) to allow cover migration for 1y. It can then be removed from Master entirely.